src/RecoveryServices/RecoveryServices.Backup.Autorest/docs/Edit-AzRecoveryServicesBackupSchedulePolicyClientObject.md

```diff
@@ -0,0 +1,454 @@
+---
+external help file:
+Module Name: Az.RecoveryServices
+online version: https://learn.microsoft.com/powershell/module/az.recoveryservices/edit-azrecoveryservicesbackupschedulepolicyclientobject
+schema: 2.0.0
+---
+
+# Edit-AzRecoveryServicesBackupSchedulePolicyClientObject
```

Comment:
The verb for this cmdlet is `Edit`. For PowerShell, `Edit` means modifying existing data by adding or removing content. Can you please confirm if it fits your scenario? Otherwise, can we use `Update` or other suitable verbs? You can refer to https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.3. Same for the other Edit-* cmdlet.

Reply:
Yes these commands are use to modify the Policy by adding/removing/modifying the settings